ngraph/core/src/op/util/precision_sensitive_attribute.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/util/precision_sensitive_attribute.hpp"
+
```

Comment:
the same comment is to incorporate compression related attributes and routines into the single files

Reply:
Same comment